examples/staged/README.md

```diff
@@ -0,0 +1,54 @@
+# About this project
+This is a sample repo that shows the evolution of an Unleash installation from simple to a more complex one.
+
+If you just want to look at what Unleash managed with Terraform looks like, check out [stage 4](./stage_4)
+
+## How to use this repo
+### Prerequisites
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.13
+- [Docker](https://docs.docker.com/get-docker/) >= 20.10.7
+- [Docker compose V2](https://docs.docker.com/compose/cli-command/) >= 2.0.0
+- [Bash](https://www.gnu.org/software/bash/) >= 5.1.8
+- Access to [Unleash Enterprise](https://www.unleash-hosted.com/) docker image
+
+### Running the steps
+A makefile makes it easy to run the steps. The steps are numbered and can be run in order. The steps are also described in the sections below.
+
+By default it will leave an instance of Unleash running at localhost:4242 so you can check the results. If you want to clean up the docker containers, run `make clean`.
```

Comment:
```suggestion
By default it will leave an instance of Unleash running at http://localhost:4242 so you can check the results. If you want to clean up the docker containers, run `make clean`.
```